app/pages/ProjectsPage.tsx

```diff
@@ -121,6 +123,17 @@ export function ProjectsPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Folder24Icon />}>Projects</PageTitle>
+        <ContextualDocsModal
+          heading="Projects"
+          icon={<Folder16Icon />}
+          summary="Projects are logical containers for managing compute, storage and network resources within a silo."
+          links={[
+            {
+              href: links.projectsDocs,
+              linkText: 'Managing Projects',
+            },
```

Comment:
I was thinking these feel a little noisy, and it occurred to me — instead of putting just the href in the `links` object, you could put the whole thing. Then this would be as neat as

```ts
links={[docLinks.projects]}
```